lib/Backend/SymTable.h

```diff
@@ -9,21 +9,25 @@ class SymTable
     typedef JsUtil::Pair<SymID, Js::PropertyId> SymIdPropIdPair;
     typedef JsUtil::BaseDictionary<SymIdPropIdPair, PropertySym *, JitArenaAllocator, PrimeSizePolicy> PropertyMap;
     typedef JsUtil::BaseDictionary<Js::PropertyId, BVSparse<JitArenaAllocator> *, JitArenaAllocator, PowerOf2SizePolicy> PropertyEquivBvMap;
-public:
-    static const int    k_symTableSize = 8192;
-    Sym *               m_table[k_symTableSize];
-    PropertyMap *       m_propertyMap;
-    PropertyEquivBvMap *m_propertyEquivBvMap;
 
 private:
+	static const int    k_symTableSize = 8192;
+	static const int    k_maxImplicitParamSlot = 4;
```

Comment:
k_ is for constants.  Originates from Hungarian notations.

---
In reply to: [73779694](https://github.com/Microsoft/ChakraCore/pull/1373#discussion_r73779694) [](ancestors = 73779694)